v3.23/backports/py3-rosdistro/force-to-enable-alpine-platform.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/src/rosdistro/distribution_file.py b/src/rosdistro/distribution_file.py
+index c330329..90ad9b1 100644
+--- a/src/rosdistro/distribution_file.py
++++ b/src/rosdistro/distribution_file.py
+@@ -74,6 +74,21 @@ class DistributionFile(object):
+                     assert os_code_name not in self.release_platforms[os_name], "Distribution '%s' specifies the os_code_name '%s' multiple times for the os_name '%s'" % (self.name, os_code_name, os_name)
+                     self.release_platforms[os_name].append(os_code_name)
+ 
++        # Tweak for alpine platform
++        self.release_platforms['alpine'] = []
++        if name == 'kinetic':
++            self.release_platforms['alpine'].append('3.7')
++        if name == 'melodic':
++            self.release_platforms['alpine'].append('3.8')
++        if name == 'noetic':
++            self.release_platforms['alpine'].append('3.11')
++            self.release_platforms['alpine'].append('3.14')
++            self.release_platforms['alpine'].append('3.17')
++            self.release_platforms['alpine'].append('3.20')
++        if name == 'humble':
++            self.release_platforms['alpine'].append('3.17')
++            self.release_platforms['alpine'].append('3.20')
```

Comment:
Should we append `3.23` for `noetic` and `humble`?

Reply:
We should probably also add `jazzy`.

Reply:
@Taka-Kazu will you port humble to 3.23?